services/QuillLMS/lib/tasks/learn_worlds.rake

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
```

Comment:
This rake task is deletable after a single run

Reply:
Is it work putting in `lib/tasks/temporary` since it's intended to run once?